bk-config/README.md

```diff
@@ -0,0 +1,43 @@
+
+This container 
+
+
+```bash
+
+docker build -t sagecontinuum/bk-config .
+
+```
+
+
+
+# Create CA and beekeeper key-pairs and certificates and registration key (NOT registration cert)
+
+```bash
+docker run -ti --rm --network beekeeper_default --name bk-config -v beekeeper-config_bk-secrets:/usr/lib/sage/ sagecontinuum/bk-config
```

Comment:
don't you need to have run `beekeeper` atleast once for the `--network beekeeper_default` to work? (when i ran through this on my machine, when I hadn't dont the `docker-compose up` yet this returned an error.

```
docker: Error response from daemon: network beekeeper_default not found.
```

In fact, why do you need the `--network` at all?

Reply:
that was just an artifact, I removed that, thx

Reply:
I think your README still has the `--network` part it for this command. unless i am confused.

Reply:
I mixed the line up and removed the network reference from another line (where the network also was not necessary). Should be gone now.